chevah/geck
# Add support for running without virtualenv.

Scope
=====

This updated the build process so that we can run it without virtualenv. Manly to make it easy to run on Windows.

Why we got into this (5 whys)
=============================

* The migration to Read The Docs was done in a hurry


Changes
=======
Add paver.sh and pavement
Update the dev documentation.


How to try and test the changes
===============================

reviewers: @hcs0 @lgheorghiu @dumol 

Check that you can clean and run the project based on the documentation.

Thanks!
